File: 2021/1/2.rb
content = File.read('input')
              .split("\n").map(&:to_i)

content = content[0..-3].zip(content[1..-2], content[2..-1]).map(&:sum)

p(content[0..-2].zip(content[1..-1])
  .map { |p| p[0].to_i - p[1].to_i }
  .select(&:negative?).length)

File: 2022/day1/2.py
import numpy as np

f = open("test.txt", "r")
input = f.read()
elves = input.split("\n\n")
elves = [[int(snack) for snack in elf.split("\n")] for elf in elves]
elves = [np.sum(elf) for elf in elves]
elves.sort()
elves = elves[-3:]

print(np.sum(elves))